packages/css/src/design-tokens/colors.css

```diff
@@ -34,4 +34,5 @@
   --vtmn-color_info: #007dbc;
   --vtmn-color_warning: #ff600a;
   --vtmn-color_danger: #e32c26;
+  --vtmn-color_transparent: transparent;
```

Comment:
I'm a bit confused about that transparent var definition.

Seems a bit 'overkill' as it doesn't really need to be set in a custom property.

And it's declared in the config file but not used at some points :

![image](https://user-images.githubusercontent.com/22882255/103521417-019ce180-4e79-11eb-9bc3-8a02c5d74cc5.png)

Reply:
(we @apply a `vtm-bg-transparent` rule set but we set border-color as straight `transparent`value )

Reply:
Yes you're right.
I've setted this in order to have classes generated by Tailwind. Instead of using `background-color: transparent`, I can use ".bg-transparent"
What do you think?

Reply:
Once again I'm still not "tailwind-literate" enough to really understand where vars come and go

but what I meant is, about transparency value : 
Can we imagine going "full tailwind"

```css

/* Ghost button */
.vtmn-btn_variant--ghost {	 
  @apply vtmn-border-transparent
    vtmn-bg-transparent
    vtmn-text-brand-digital;
}
```
or not at all
```css
/* Ghost button */
.vtmn-btn_variant--ghost {	 
  --btn-border-color: transparent;
  --btn-background-color: transparent;
  
  @apply vtmn-text-brand-digital;
}
```

About variable : 
Either way, transparent is transparent :) 
I'm not sure it's useful to put it in a var (but that's opiniated, I concede ^^)

Reply:
You're right. We can imagine a not "full tailwind" world for values like transparent that are always transparent (: